src/main/java/com/synopsys/integration/alert/channel/email/web/EmailAddressSelectOption.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * blackduck-alert
+ *
+ * Copyright (c) 2020 Synopsys, Inc.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.synopsys.integration.alert.channel.email.web;
+
+import com.synopsys.integration.alert.common.rest.model.AlertSerializableModel;
+
+public class EmailAddressSelectOption extends AlertSerializableModel {
+    private String emailAddress;
+    private Boolean optOut;
```

Comment:
We may want to discuss not exposing the `optOut` field, since it currently does not serve a functional purpose.

Reply:
We can annotate it with `JSONIgnore` so that Jackson will not serialize it or if we aren't going to have optOut then we should remove it.

Reply:
Alert is already serializing the optOut from the ProviderUserModel class. That's what the custom endpoint did in the past.  Since the custom endpoint dealt with email addresses I created this object to represent the email address data.